content/agent/basic_agent_usage/amazonlinux.md

```diff
@@ -2,86 +2,152 @@
 title: Basic Agent Usage for Amazon Linux
 kind: documentation
 platform: Amazon Linux
-servicename: /etc/init.d/datadog-agent
-serviceinfoname: /etc/init.d/datadog-agent info
-configdirectory: /etc/dd-agent/
-logdirectory: /var/log/datadog/
-supervisorlog: /var/log/supervisor/datadog-supervisord.log
 aliases:
     - /guides/basic_agent_usage/amazonlinux/
 ---
 
 ## Overview
 
-This page outlines the basic functionality of the Datadog Agent.
+This page outlines the basic functionality of the Datadog Agent for Amazon linux.  
 If you haven't installed the Agent yet, instructions can be found
 [in the Datadog agent integration page](https://app.datadoghq.com/account/settings#agent/aws)
 
-The process to upgrade from the previous version of the agent is to re-run the installation.
+## Commands
 
-## Starting and Stopping the Agent
+Datadog Agent has some commands and only the _lifecycle commands_ (i.e. `start`/`stop`/`restart`/`status` on the Agent service) should be run with `sudo service`/`sudo initctl`/`sudo systemctl`, all other commands need to be run with the `datadog-agent` command.
```

Comment:
this can be tighter / more terse.

Reply:
how?

Reply:
I'd agree that it reads awkward. Will need to think about a specific recommendation.

Reply:
Given the table below, is this qualifying sentence even needed?

Reply:
I'm buying it !